README.md

```diff
@@ -2,13 +2,11 @@
 ## About ARC Voucher Service/API
 ARCV Service is the service portal and API for ARCV Market.
 
-[![Build Status](https://travis-ci.org/neontribe/ARCVMarket.svg?branch=0.2/release)](https://travis-ci.org/neontribe/ARCVMarket.svg?branch=master)
```

Comment:
Why remove?

Reply:
becuase it relates to the market app, and this is the service/store repo

Reply:
Fair!